.travis-push.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+setup_git() {
+  git config --global user.email "ci@coursier.iMadeThisUp"
+  git config --global user.name "Coursier Travis CI"
+}
+
+commit_website_files() {
+  git add README.md
+  git commit -m "Travis CI: readme update - $TRAVIS_BUILD_NUMBER"
+}
+
+upload_files() {
+  git remote add origin https://${GH_TOKEN}@github.com/coursier/apps.git > /dev/null 2>&1
```

Comment:
For this to work, you'll need a Github token set up. You may already have this, and I'll just need to update the variable here.

Reply:
Just added one.